doc/api/next_api_changes/deprecations/20585-EP.rst

```diff
@@ -0,0 +1,19 @@
+RectangleSelector and EllipseSelector
```

Comment:
I would give this some context so that it does not appear like a random change of many parameters, e.g.

```
Unification of Selector API
~~~~~~~~~~~~~~~~~~~~~~~~~~~
The API for Selector widgets has been unified to use

- `props` for the properties of the Artist representing the selection.
- `handle_props` for the Artists representing handles for modifying the selection.
- `grab_range` for the maximal tolerance to grab a handle with the mouse.

This affects the following parameters and attributes:
```

+ Following section headers with 1 level reduced.

Reply:
Done in 00444e4.